packages/firebase_auth/lib/src/firebase_auth.dart

```diff
@@ -241,15 +241,20 @@ class FirebaseAuth {
   /// [codeAutoRetrievalTimeout] Optional callback.
   ///   It will trigger when SMS auto-retrieval times out and provide a
   ///   [verificationId].
-  Future<void> verifyPhoneNumber({
-    @required String phoneNumber,
-    @required Duration timeout,
-    int forceResendingToken,
-    @required PhoneVerificationCompleted verificationCompleted,
-    @required PhoneVerificationFailed verificationFailed,
-    @required PhoneCodeSent codeSent,
-    @required PhoneCodeAutoRetrievalTimeout codeAutoRetrievalTimeout,
-  }) async {
+  ///
+  /// [linkCredentials] Default false.
+  ///   Set to true when phone number verification is used to link a phone number to an existing user who is
+  ///   currently logged in
+
+  Future<void> verifyPhoneNumber(
+      {@required String phoneNumber,
+      @required Duration timeout,
+      int forceResendingToken,
+      @required PhoneVerificationCompleted verificationCompleted,
+      @required PhoneVerificationFailed verificationFailed,
+      @required PhoneCodeSent codeSent,
+      @required PhoneCodeAutoRetrievalTimeout codeAutoRetrievalTimeout,
+      bool linkCredentials = false}) async {
```

Comment:
Could you please add back the comma (,) at the end of the array setup and reformat?